outbox-kafka-spring/src/test/java/one/tomorrow/transactionaloutbox/service/OutboxLockServiceIntegrationTest.java

```diff
@@ -1,5 +1,5 @@
-/**
- * Copyright 2022 Tomorrow GmbH @ https://tomorrow.one
+/*
+ * Copyright 2023 Tomorrow GmbH @ https://tomorrow.one
```

Comment:
I think this should not be touched, the year stands for the year when this file got publicly released.

It also breaks the `licenseTest` task, see the failed build... To relax the license plugin for new files in the future can probably be kept out of this here.